src/astrobleme/logic/History.java

```diff
@@ -0,0 +1,4 @@
+package astrobleme.logic;
+
+public class History {
```

Comment:
What history will you be storing here? I assume this is for score list?

Reply:
It is meant for collecting the snake movement in the past. I am not sure when to start working on that.